javascript/app.js

```diff
@@ -0,0 +1,70 @@
+console.log("up and running");
+
+
+// 1. A parameter is a varible that is used to declare a function within the (), while an argument consists of the actually values you pass into a function when you invoke that function i.e. const someFunction (parameter) => {} vs. someFunction(argument);
+// 2. Return gives the function a value within your program when the program is run while console.log will display the result of the invoked function within your console
```

Comment:
console.log() will display whatever you pass to it whether that's the "function result" or anything else.  (i suspect you're well aware of this now)